trin-beacon/src/network.rs

```diff
@@ -1,6 +1,5 @@
-use std::sync::Arc;
-
 use parking_lot::RwLock as PLRwLock;
+use std::sync::Arc;
```

Comment:
*nitpick* but according to our guidelines, the `std` module imports are supposed to be in an isolated 1st grouping, before the external dependency imports grouping, followed by the internal dependency imports grouping. Any specific reason you changed that here (and in other places in this pr)?